tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -96,10 +100,11 @@ def _get_tree(self, result: Mapping[str, Any]) -> Any:
         return result["contexts"]
 
     @command
-    def handle_user_prompt(self,
-                           context: str,
-                           accept: Optional[bool] = None,
-                           user_text: Optional[str] = None) -> Mapping[str, Any]:
+    def handle_user_prompt(
+            self,
+            context: str,
```

Comment:
It would be good if you wouldn't change unnecessary details to make reviewing easier. Especially these changes add a third way of formatting. I would suggest that we revert that and maybe the `capture_screenshot` ones as well.